### DISCUSSION TITLE
can't initialize iptables table `mangle'

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [v2rayA/v2rayA#1591] [Q&A]
今天v2rayA无法启动，显示错误：`failed to start v2ray-core: not support "tproxy" mode of transparent proxy: ExecCommands: iptables -w 2 -t mangle -N TP_OUT iptables v1.8.10 (legacy): can't initialize iptables table `mangle': Table does not exist (do you need to insmod?) Perhaps iptables or your kernel needs to be upgraded. : exit status 3`

如果我运行：`iptables -w 2 -t mangle -N TP_OUT`会得到
`Fatal: can't open lock file /run/xtables.lock: Permission denied`

系统：archlinux+v2ray 5.18.0+v2rayA 2.2.6.3

**Comment by See-Night** [comment]
我也遇到这个问题了

**Comment by Firestar-Reimu** [comment]
重启之后莫名其妙就好了

**Comment by puji4810** [comment]
我也遇到了，看完这个重启了一下，莫名其妙好了，我还以为是内核里面缺了什么模块
